plugins/rbac/src/components/CreateRole/CreateRolePage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Content, Header, Page } from '@backstage/core-components';
+import { catalogEntityReadPermission } from '@backstage/plugin-catalog-common/alpha';
+import { RequirePermission } from '@backstage/plugin-permission-react';
+
+import { CreateRoleForm } from './CreateRoleForm';
+
+export const CreateRolePage = () => {
+  return (
+    <RequirePermission
+      permission={catalogEntityReadPermission}
+      resourceRef={catalogEntityReadPermission.resourceType}
```

Comment:
Should check for catalogEntityCreatePermission

Reply:
> > > > @parvathyvr @debsmita1 for now I have added the following permission checks on accessibility of `Create` role button:
> > > > 
> > > > * policy.entity.create
> > > > * catalog.entity.read
> > > > * catalog.entity.create
> > > > 
> > > > If the user has all the above permissions then they can go ahead and create the role and policies to it, otherwise the button is disabled. LMK if anything else is required here.
> > > 
> > > 
> > > @divyanshiGupta `catalog.entity.create` this won't be necessary, we only want to fetch the users and groups
> > 
> > 
> > @debsmita1 I have added `catalog.entity.create` because during role creation we make an api call for creating the role as well. LMK if we dont need this check for that or `policy.entity.create` is sufficient?
> 
> for creating the role we need policy.entity.create

@debsmita1 I am confused. Do we need catalog.entity.create for creating roles or not?

Reply:
Sorry, my bad
I was looking for policyEntityCreate & catalogEntityRead .Found the 2 places where you have added the permission checks